### DISCUSSION TITLE
ADO2GH for TFVC/VSTS Source

### DISCUSSION THREAD

**Discussion by mark-evans-tfs** [github/gh-gei#1090] [Q&A] [answered]
It seems like the Azure DevOps to GitHub Enterprise Importer tool only supports the git variant of source control in DevOps - could I confirm that is the case please? I've been through the docs but have been unable to find a definitive statement. The tool itself asks for an ADO Repo and adds a /_git/ when searching for the source.

Cheers

**Comment by dpmex4527** [comment, marked as answer]
This is correct. ADO2GH only supports git variant from Azure DevOps. Microsoft does have [tooling](https://learn.microsoft.com/en-us/devops/develop/git/migrate-from-tfvc-to-git) to convert TFVC/VSTS into git repos so my suggestion would be to first migrate source to git and then use ADO2GH to migrate the git source into GitHub.com.
